src/main/java/com/dev/cinema/config/WebConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.dev.cinema.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"com.dev.cinema.controllers"})
+public class WebConfig {
+    @Bean
+    public InternalResourceViewResolver getResolver() {
+        InternalResourceViewResolver resolver
+                = new InternalResourceViewResolver();
+        resolver.setViewClass(JstlView.class);
+        resolver.setPrefix("/WEB-INF/views/");
```

Comment:
You don't have any views in your project now.

Reply:
Done